dev/source/docs/ros2.rst

```diff
@@ -52,6 +52,20 @@ Now update all dependencies:
     rosdep update
     source /opt/ros/humble/setup.bash
     rosdep install --from-paths src --ignore-src
+    # Ignore the rosdep warnings at the end if any
+
+Installing build dependencies:
+
+.. code-block:: bash
+    
+    sudo apt install default-jre
+    cd ~/ros2_ws
+    git clone --recurse-submodules https://github.com/ardupilot/Micro-XRCE-DDS-Gen.git
+    cd Micro-XRCE-DDS-Gen
+    ./gradlew assemble
+    echo "export PATH=$PATH:$PWD/scripts" >> ~/.bashrc
```

Comment:
Please replace this with what is in the README. This change you made is wrong, and stores your current `$PATH` in the ~/.bashrc.

Also, you dont need to create  a new PR every time you push changes.

Reply:
It is doing the same thing as mentioned in the [README](https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_DDS#installing-build-dependencies). Instead of manually entering the path to the `Micro-XRCE-DDS-Gen/scripts`, it is just using the $PWD from the env and so person does not have to change anything and can just copy paste it while following the process.

Reply:
But it doesn't do the same thing. Run the way I wrote it,and then run the way you did it. Then open your bashrc, and you will see the results are not the same. I totally get what you are trying to do; it just doens't work.

Reply:
If you can find the right syntax to get it to behave the same, that's fantastic, but I'm not willing to accept the PR with the current different behavior.

Reply:
Done :+1: